static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +75,41 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
+            return detect.isIOS() || detect.isAndroid();
```

Comment:
Yes, this is specifically the check I want as these platforms block autoplay at the OS level.